src/project-3.js

```diff
@@ -87,6 +118,13 @@ const addCalculateDiscountPriceMethod = (storeItem) => {
   // discountPrice = 20 - (20 * .2)
   // Make sure you return storeItem after adding the method to it
   // hint: arrow functions don't bind a this
+  storeItem.calculateDiscountPrice = function() {
```

Comment:
To use the arrow function (since `this` isn't bound to the `storeItem` object), you can use the following:

```
storeItem.calculateDiscountPrice = () => {
  const discount = storeItem.price * storeItem.discountPercentage;
  return storeItem.price - discount;
};
```